src/docs/home/Home.scss

```diff
@@ -1,3 +1,6 @@
+// TODO: Remove import once stylesheet is moved off of component
```

Comment:
Do we need a ticket for this / to add this to one of the existing refactoring tickets?